config/cron-make-nightly-tarball.pl

```diff
@@ -121,6 +124,31 @@ sub verbose {
 print OUT $config;
 close(OUT);
 
+# git describe output looks like: ${tag}-${count}-g${commitid}
+# Fedora guidelines want RPM release tag to look like:
+# Release: 0.${date}git${commitid}
+my @gdsplit = split(/-/, $gd);
+my $commitid = substr($gdsplit[-1], 1);
```

Comment:
Might be better to use a regexp (untested -- just typed in github web UI, so test this before accepting):

```perl
$gd =~ m/-g(\w+?)$/;
my $commitid = $1;
```